src/Containers/Containers.jl

```diff
@@ -0,0 +1,204 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# JuMP
+# An algebraic modeling language for Julia
+# See http://github.com/JuliaOpt/JuMP.jl
+#############################################################################
+
+"""
+    Containers
+
+Module defining containers `DenseArray` and `SparseArray` that behaves as
```

Comment:
Nit: This doesn't accurately describe the code because `DenseArray` isn't part of the module yet.